src/extension/src/coreTemplateStudio.ts

```diff
@@ -1,16 +1,16 @@
-import fetch, { Response } from "node-fetch";
-import * as signalR from "@aspnet/signalr";
-import * as portfinder from "portfinder";
 import * as vscode from "vscode";
 import * as path from "path";
 import * as os from "os";
 import * as fs from "fs";
 
-import { ChildProcess, execFile } from "child_process";
+import { ChildProcess, spawn } from "child_process";
 import { CONSTANTS } from "./constants";
-import { GenerateCommand } from "./signalr-api-module/generateCommand";
-import { SyncCommand } from "./signalr-api-module/syncCommand";
-import { ICommandPayload } from "./signalr-api-module/commandPayload";
+import { ICommandPayload } from "./types/commandPayload";
+import { IGenerationPayloadType } from "./types/generationPayloadType";
+import { EventEmitter } from "events";
+import { IEngineGenerationPayloadType } from "./types/engineGenerationPayloadType";
+
+class CliEventEmitter extends EventEmitter { }
```

Comment:
Considering that `CliEventEmitter `is in no way different from `EventEmitter `, what was the reason to create a new type here?

Reply:
Didn't work using `EventEmitter` directly for some reason

Reply:
If you look at `src/extension/node_modules/@types/node/index.d.ts` line 938, you can see `EventEmitter` defined and exported as a class.

Have you tried defining `private cliEvents: EventEmitter;` and `this.cliEvents = new EventEmitter();` in your constructor?

If so, could you please post your error?